lib/pool.js

```diff
@@ -420,12 +438,17 @@ var pool = module.exports = function pool(options, authorizeFn) {
                 return util.addressToScript(rpcResults.validateaddress.address);
             })();
 
-            options.testnet = rpcResults.getinfo.testnet;
-            options.protocolVersion = rpcResults.getinfo.protocolversion;
+            options.testnet = rpcResults.getblockchaininfo.chain === "test";
```

Comment:
I would expect this to fail if the coin's config has `hasGetInfo` set to true since we're only calling `getblockchaininfo` if `hasGetInfo` is false or unset on lines 394-398 above.

Reply:
I added a conditional... I think that is what you meant.

Reply:
Yes sir, thank you!

Reply:
@hashtobewild @egyptianbman This caused the stratum-pool to crash on startup after this change.

Reply:
@egyptianbman @hashtobewild this change caused the pool to crash when trying to pull in daemon stats for the first time.

Reply:
@nettts I'll run a test and create a PR with a fix. What configs are you running?